Frameworks/include/CACompositor.h

```diff
@@ -148,4 +160,5 @@ CA_EXPORT CACompositorInterface* GetCACompositor();
 CA_EXPORT void SetCACompositor(CACompositorInterface* compositorInterface);
 
 CA_EXPORT bool CASignalDisplayLink();
```

Comment:
Could we move these global vars/defines to the top of the file? It's really a pain tracking them down when they are scattered within the body of a file.